docs/docs/start/docker_dev.md

```diff
@@ -115,6 +118,49 @@ This command launches the remaining containers:
 !!! success "Check Connection"
     Check that the server is running at [http://localhost:8000](http://localhost:8000). The server may take a few minutes to be ready.
 
+## Running commands in the container
+
+Using `docker compoase run [...]` commands creates a new container to run this specific command.
+This will eventually clutter your docker with many dead containers that take up space on the system.
+
+You can access the running containers directly with the following:
+```
+docker exec -it inventree-dev-server /bin/bash
+```
+You then run the following to access the virtualenv:
+```
+source data/env/bin/activate
+```
+
+This sets up a bash terminal where you can run `invoke` commands directly.
+
+!!! warning "Tests"
+    It is not recommended to run `invoke test` in your currently active inventree-dev-server container.
+
+### Cleaning up old containers
+
+To clean out old containers, follow this guide:
+
+Run the following command:
+```
+docker container ls --filter name=inventree*
+```
+This should yield 3 lines of output looking something like this:
+
+```
+CONTAINER ID   IMAGE                 COMMAND                  CREATED       STATUS      PORTS                    NAMES
+c2f4043bb98d   inventree-dev-image   "/bin/bash ./docker/…"   3 days ago    Up 3 days                            inventree-dev-worker
+4f8e1b25aeba   inventree-dev-image   "/bin/bash ./docker/…"   3 days ago    Up 3 days   0.0.0.0:8000->8000/tcp   inventree-dev-server
+d674d203ae87   postgres:13           "docker-entrypoint.s…"   4 weeks ago   Up 4 days   5432/tcp                 inventree-dev-db
+```
+
+You should have 1 inventree-dev-worker, 1 inventree-dev-server, and 1 inventree-dev-db in that list.
+If, however, you don't have these, refer to [Start Docker Containers](#Start Docker Containers) and then return here.
+
+```
+docker container prune --filter label="com.docker.compose.project=inventree" --filter label="com.docker.compose.oneoff=True"
+```
```

Comment:
Will the recommended commands only touch development containers? Is there a risk that a prod instance is also removed?

Reply:
In my testing, `com.docker.compose.oneoff` is only set to `True` on containers that are executed with `docker compose run [image] [cmd]`. The containers running off of `docker compose up -d` have this flas set to `False`

Reply:
So if somebody starts their prod instance with run (maybe a scheduled task that opens a shell for that - I have seen that) it is also gone? Can we filter for explicitly dev instances? Else there should be a big warning above the commands.

Reply:
Are you able to get me an output from 'docker inspect [containerid]' from a prod setup?

Reply:
Im not sure if we can DeTeXt prod/dev 100%, and even if we could, I'm also not sure if we should build such a big oneliner beginners should just copy and paste. And also Dev setups can also contain data that is not in a volume because you tested something. So my recommendation would be to only add this command with a very big disclaimer what can happen.